SWLOR.Game.Server/Service/CombatPoint.cs

```diff
@@ -270,16 +289,32 @@ public static void ClearRecentEnemyLevel(uint player)
             DeleteLocalInt(player, "COMBAT_POINT_LAST_NPC_LEVEL");
         }
 
+        /// <summary>
+        /// Retrieves the count of creatures tagged by a given player.
+        /// </summary>
+        /// <param name="player">The player to check for tagged NPCs.</param>
+        /// <returns>
+        /// Number of creatures tagged by the player. 
+        /// 0 is none, -1 if the player is not initialized in the playerToCreatureTracker.
+        /// </returns>
+        public static int GetTaggedCreatureCount(uint player)
+        {
+            if (!_playerToCreatureTracker.ContainsKey(player))
+                return -1;
+
+            return _playerToCreatureTracker[player].Count;
+        }
+
         /// <summary>
         /// Adds a combat point for a player to all NPCs s/he is currently tagged on.
         /// </summary>
         /// <param name="player">The player to receiving the point.</param>
         /// <param name="skill">The skill to associate with the point.</param>
         /// <param name="amount">The number of points to add.</param>
-        /// <returns>True if at least one creature is tagged, false otherwise.</returns>
+        /// <returns>Number of creatures tagged by the player, -1 if the tracker has expired or is unavailable.</returns>
```

Comment:
Sorry - can you remove this comment? No longer relevant because of the new method you added.